packit/api.py

```diff
@@ -1730,3 +1732,45 @@ def init_source_git(
             ignore_missing_autosetup=ignore_missing_autosetup,
         )
         sgg.create_from_upstream()
+
+    def run_mock_build(
+        self,
+        root: str,
+        srpm_path: Path,
+    ) -> List[Path]:
+        """
+        Performs a mock build with given SRPM and root.
+
+        Args:
+            root: Name of the chroot or path to the mock config.
+            srpm_path: Path to the SRPM to be built.
+
+        Returns:
+            List of paths to the built RPMs.
+        """
+        cmd = ["mock", "--root", root, str(srpm_path)]
```

Comment:
Is the `root` argument required?

Reply:
it shouldn't be required, there is a default (which usually points to rawhide)

Reply:
> Is the `root` argument required?

it is not, if it's not specified it defaults to (surprisingly) `default`… IMO it is easier to have `default` as the default root than handling explicitly specified root as a special case

```
       -r CONFIG, --root=CONFIG
              Uses specified chroot configuration as defined in ~/.config/mock/<CONFIG>.cfg or /etc/mock/<CONFIG>.cfg.  Optionally if CONFIG ends in '.cfg', it is interpreted as full path to config file. If none specified, uses the chroot config linked to by /etc/mock/default.cfg.
```

Reply:
I see have a default for the CLI, but not for this method and I am wondering if we don't want to use `default` default also here...

Reply:
not sure if there is a good reason for that, but done :)